### PR TITLE
Add Nucleotide Count Test Generator #416

### PR DESCRIPTION
**Pull request by InKahootz**
This was/is a good learning experience. I look forward to feedback and doing more work.
I tried to become as familiar as possible with the code base to come up with an elegant solution, though I didn't see a way to easily test instance properties. 

As explained further in my commit message, I took the liberty to remove the Count method from the exercise. It was previously needed in he the old unit testing but with the canonical data based unit test, it was pruned.

### CODE REVIEW

**Comment by jpreese** [comment]
Awesome @InKahootz! We always welcome the help 👍 

As a heads up, if you haven't figured out already, the broken build is due to the fact that the class name was changed and the `Example.cs` was not changed as well. That should get Travis to go green for you.

You can also run powershell .\build.ps1 (for windows) or build.sh (for linux) to test your build before pushing to your branch if you so desire.

**Comment by InKahootz** [comment]
Ah, I forgot `Example.cs`. Doesn't show up in VS. I'll get that fixed right quick.

File: generators/Exercises/NucleotideCount.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+﻿using Generators.Input;
+using Generators.Output;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Generators.Exercises
+{
+    class NucleotideCount : Exercise
```
Comment (ErikSchierboom) [inline generators/Exercises/NucleotideCount.cs]: Could you add the `public` modifier (for consistency)?

**Comment by InKahootz** [comment]
Thanks! I'll start looking for my next issue.
One question. Is there no way to test instance/static properties on a class without writing a custom template?

**Comment by ErikSchierboom** [comment]
@InKahootz There should be. I'll look into it later. I'll merge when the CI is green.

**Comment by ErikSchierboom** [comment]
Merged! Thanks a lot @InKahootz! :tada:

**Pull request merged**